### PR TITLE
feat: trigger hover after scroll

### LINKED ISSUE #2222: 配置了interaction: { enableCopy: true, hoverHighlight: true }，滚动之后，自动触发当前鼠标所在单元格的high linght效果🙏
### 🖋 Description

配置了interaction: { enableCopy: true, hoverHighlight: true }，滚动之后，自动触发当前鼠标所在单元格的high linght效果，

### 🏞 What problem does this feature solve

优化了交互体验，现状是，需要鼠标移动一下，才能触发high linght效果

### 🧐 Suggest the API

| name | type | default | description |
| ---- | ---- | ------- | ----------- |
| -    | -    | -       | -           |

### PR DESCRIPTION
**Pull request by tyr1dev**
### 👀 PR includes

✨ Feature

- [x] New feature

🎨 Enhance

- [ ] Code style optimization
- [ ] Refactoring
- [ ] Change the UI
- [ ] Improve the performance
- [ ] Type optimization

🐛 Bugfix

- [ ] Solve the issue and close #0

🔧 Chore

- [ ] Test case
- [ ] Docs / demos update
- [ ] CI / workflow
- [ ] Release version
- [ ] Other ()

### 🖼️ Screenshot

| Before | After |
| ------ | ----- |
| ❌      | ✅     |

### 🔗 Related issue link

close #2222

### 🔍 Self-Check before the merge

- [ ] Add or update relevant docs.
- [ ] Add or update relevant demos.
- [x] Add or update test case.
- [ ] Add or update relevant TypeScript definitions.

### CODE REVIEW

**Review by lcx-seima — COMMENTED** [review]
LGTM

**Comment by lcx-seima** [comment]
线下跑过测试了？线上环境不同，是可能挂的

**Comment by tyr1dev** [comment]
> 线下跑过测试了？线上环境不同，是可能挂的

线下跑过了，最近几次经常出现 CI 重新跑一次又过了的情况，如果可以的话可以帮我重新触发一下 CI

File: packages/s2-core/__tests__/spreadsheet/scroll-spec.ts [hunk 1]
```diff
@@ -390,6 +390,55 @@ describe('Scroll Tests', () => {
...
+      },
+    },
+  ])(
+    'should trigger hover cells when hover cells after scroll by %o',
+    async ({ offset }) => {
+      s2.facet.cornerBBox.maxY = -9999;
+      s2.facet.panelBBox.minX = -9999;
+      s2.facet.panelBBox.minY = -9999;
+
+      const mousemoveEvent = new MouseEvent('mousemove', {
```
Comment (lcx-seima) [inline packages/s2-core/__tests__/spreadsheet/scroll-spec.ts:416]: 构造 mouse event 的时候，这样试试？
```javascript
const bbox = s2.getCanvasElement().getBoundingClientRect();
const mousemoveEvent = new MouseEvent('mousemove', {
  clientX: bbox.left + 100,
  clientY: bbox.top + 100,
});
```

**Review by lcx-seima — DISMISSED** [review]

**Comment by lcx-seima** [comment]
本次 pr 后交互行为有变化，比如滚动后会有连续的 hover，tooltip 也会出现。
master 版本下为控制使用风险，增加一个 options 来开/闭行为吧

File: s2-site/examples/interaction/basic/demo/meta.json [hunk 2]
```diff
@@ -75,6 +75,14 @@
...
       },
       "screenshot": "https://gw.alipayobjects.com/zos/antfincdn/Lb0%26u6LtAu/reset.gif"
+    },
+    {
+      "filename": "hover-after-scroll.ts",
+      "title": {
+        "zh": "滚动后自动触发悬停状态",
+        "en": "Trigger hover after scroll"
+      },
+      "screenshot": "https://gw.alipayobjects.com/zos/antfincdn/l23NpRrPmF/hover.gif"
```
Comment (tyr1dev) [inline s2-site/examples/interaction/basic/demo/meta.json:85]: 替换截屏链接

**Comment by tyr1dev** [comment]
hover-after-scroll

请帮我上传一下截图然后替换一下链接

File: s2-site/examples/interaction/basic/demo/meta.json (hunk 2 above)
Comment (lcx-seima) [reply s2-site/examples/interaction/basic/demo/meta.json:85]: > hover-after-scroll
> 
> 请帮我上传一下截图然后替换一下链接

https://mdn.alipayobjects.com/huamei_qa8qxu/afts/img/A*NbG7SakRzXUAAAAAAAAAAAAADmJ7AQ/original

**Review by lcx-seima — APPROVED** [review]

**Pull request merged**